src/website2/.env.sample

```
@@ -1,5 +1,11 @@
-NEXT_PUBLIC_SITE_URL=https://your-site-url.com
-NEXT_PUBLIC_API_URL=https://external-api-url
-NEXT_PUBLIC_OPENCAGE_API_KEY=
+# Server-side only variables (not exposed to client)
+API_URL=
+OPENCAGE_API_KEY=
+API_TOKEN=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The API_TOKEN key should go before the API_URL key

(UnorderedKey)